### DISCUSSION TITLE
Argo CD Operator v0.2.0 is out !!

### DISCUSSION THREAD

**Discussion by iam-veeramalla** [argoproj-labs/argocd-operator#546] [Show and tell]
Argo CD Operator v0.2.0 is released to operatorhub.io . Thanks everyone who has contributed for  the release. Cheers :argocd:

Version from Application Logs:
`version: v2.2.2+03b17e0, built: 2022-01-01T06:27:52Z)`

Download Operator :
https://operatorhub.io/operator/argocd-operator

Release Notes:
https://github.com/argoproj-labs/argocd-operator/releases/tag/v0.2.0

**Comment by iam-veeramalla** [comment]
Thanks !!

**Comment by felixkrohn** [comment]
Thanks for you work!
As there is some urgency to update from AroCD v2.2.2 to v2.2.4, will we see an argocd-operator v0.2.1 anytime soon? Or is there a way to use the recently fixed ArgoCD images without updating the operator itself?

**Reply by iam-veeramalla** [reply]
Hey, Yes you can update the Image using Argo CD CR.
https://argocd-operator.readthedocs.io/en/latest/reference/argocd/#image

Nevertheless, we are planning for a new release with fix version of Argo CD asap.
